java/ql/test/library-tests/dataflow/collections/options

```diff
@@ -0,0 +1 @@
+//semmle-extractor-options: --javac-args --release 21
```

Comment:
There are no other changes in this dir? What does this change accomplish?

Reply:
Removed